src/pages/LandingPage.js

```diff
@@ -47,10 +47,10 @@ class Home extends Component {
           style={{ flexDirection: "column" }}
         >
           <Typography variant="h5" align="center">
-            React Github Resume {packageJson.version}
+            Github Resume Generator {packageJson.version}
           </Typography>
           <Typography variant="body1" className="description" align="center">
-            Please enter Github profile name with case sensitive
+            Please enter Github username
```

Comment:
GitHub

Reply:
I think `name with case sensitive` is not necessary

Reply:
yes.. that's ok...  In GitHub ... H - Capital

Reply:
Okay, wait

Reply:
@sabesansathananthan need any other changes?